deploy/kubernetes/overlays/dev/controller_images.yaml

```diff
@@ -6,6 +6,12 @@ spec:
   template:
     spec:
       containers:
+        - name: csi-provisioner
+          image: gcr.io/gke-release-staging/csi-provisioner:v0.4.1-gke.0
```

Comment:
gke-release-staging is not publicly accessible, so we need to use gke-release or quay here.  Same with stable and prow-head-template (which is used in oss pull jobs)

Reply:
ah my bad, botched copy-paste job